lib/mongorito.js

```diff
@@ -866,9 +888,9 @@ exports = module.exports = Mongorito;
 
 exports.Model = Model;
 
-Object.keys(mongoskin).forEach(function (key) {
+Object.keys(mongodb).forEach(function (key) {
   if (['connect', 'version', 'db'].indexOf(key) === -1) {
-    exports[key] = mongoskin[key];
+    exports[key] = mongodb[key];
```

Comment:
Expose everything except classes like MongoClient and Db. Cursor and ObjectId should be exposed.

No need to test them, just test that those properties are exposed and that's enough.